ddtrace/tracer/option_test.go

```diff
@@ -190,3 +190,16 @@ func TestServiceName(t *testing.T) {
 		assert.Equal("", globalconfig.ServiceName())
 	})
 }
+
+func TestGlobalTag(t *testing.T) {
+	t.Run("InStatsTags", func(t *testing.T) {
```

Comment:
If you have sub-tests, they need to be more than one, otherwise, just one test is good.

Reply:
Sounds good, fixed